pom.xml

```diff
@@ -148,33 +150,17 @@
             </execution>
           </executions>
         </plugin>
-        <plugin>
```

Comment:
Not sure if the removal here is intentional, or just due to missing fetch, but was hoping to add defaults for moditect plugin (wrt where to add module-info). Not a big deal, so if it is necessary to duplicate on child poms that's ok, but wanted to know if this was necessary.
(my understanding on pluginManagement section is that settings are inherited from parent, so it is possible to augment settings on different levels)